addons/account/models/account_move.py

```diff
@@ -2320,6 +2320,25 @@ def write(self, vals):
 
         return res
 
+    def check_move_sequence_chain(self):
+        return self.filtered(lambda move: move.name != '/')._is_end_of_seq_chain()
+
+    @api.ondelete(at_uninstall=False)
+    def _unlink_forbid_parts_of_chain(self):
+        """ For a user with Billing/Bookkeeper rights, when the fidu mode is deactivated,
+        moves with a sequence number can only be deleted if they are the last element of a chain of sequence.
+        If they are not, deleting them would create a gap. If the user really wants to do this, he still can
+        explicitly empty the 'name' field of the move; but we discourage that practice.
```

Comment:
I don't see the point of adding a constraint if they can still do it?
Seems like more code for nothing much?

Reply:
Also, if we can do it, what's the point of the JS confirmation widget?

Reply:
they can only do it under specific circumstances and the point of the widget is to notify them about the consequences before they do it